### ISSUE TITLE
TrackJS use is out of quota

### ISSUE THREAD

**Issue by VictorVelarde** [CartoDB/cartodb#14890]
### Context

We currently have a license to use  [TrackJS](https://trackjs.com/) for JS errors. The 'applications' registered in that tool are:

- Builder
- Builder embed
- CartoDB central
- CartoDB editor
- CartoDB embeds
- CartoJS

It is a really useful tool, but now we need to make some changes in how we use it.

### Current

Our limits are set for **10 million page views** (with 15 days retention), but we have run several months out of that quota, as TrackJS people have gently notified us.

This TrackJS dashboard displays our metrics: https://my.trackjs.com/usage (requires FrontEnd credentials) 

So now we need to take some action on it.

We were offered an upgrade to the next level (25 million page views), but after discussing it internally, we have finally decided to reduce our use, so we can fit into the current plan.

That's mainly because of 2 reasons:

- the apps are pretty stable now
- most of the quota is consumed by 'embeds' (>=80%), and we think we can reduce safely that 'noise'

For example during the last 2 weeks:

- 6.6 million page views
  - Builder embed: 4 million
  - CartoDB embeds: 2.5 million

### Expected result
We must take some actions to we fit into the current paid plan.

We should conditionally include the library in the embeds, so we lower its page views number. For the very nature of those embeds, we think we can safely reduce the inclusion to just a 20%. Discarding the other 80% would for sure make us fit into the plan, but without losing relevant issues.

The other task is having a quick review of the noise / signal ratio in other apps, to see if we can improve there. 

So, to sum up:

- [x] Reduce the use in embeds:
  - Builder embed
  - CartoDB embeds

- [x] Quick review of TrackJS at:
  - Builder
  - CartoDB central
  - CartoDB editor
  - CartoJS

### Additional info
I'm gonna notify TrackJS people that we are working actively on this, by sharing the link to this issue.

**Comment by VictorVelarde** [comment]
Please, @CartoDB/rt-managers add this to RT, so we can start working on it asap

**Comment by VictorVelarde** [comment]
For the first time in some months, in June (2019) we're under quota limits again. 👍
